generators/liquibase/templates/gradle/liquibase.gradle.ejs

```diff
@@ -1,7 +1,10 @@
+configurations {
+    liquibaseRuntime.extendsFrom sourceSets.main.compileClasspath
```

Comment:
Only if database SQL?

Reply:
Should not be needed as the file is only created for sql databases